src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts

```diff
@@ -61,7 +61,13 @@ export class ManageAuthorisedPeopleHandler extends GenericHandler {
             buttonHref: getUrlWithCompanyNumber(constants.YOUR_COMPANIES_ADD_PRESENTER_URL, companyNumber) + constants.CLEAR_FORM_TRUE,
             cancelUrl: constants.YOUR_COMPANIES_COMPANY_AUTH_PROTECTED_CANCEL_PERSON_URL.replace(`:${constants.COMPANY_NUMBER}`, companyNumber),
             resendEmailUrl: constants.YOUR_COMPANIES_MANAGE_AUTHORISED_PEOPLE_EMAIL_RESENT_URL,
-            removeUrl: constants.YOUR_COMPANIES_AUTHENTICATION_CODE_REMOVE_URL
+            removeUrl: constants.YOUR_COMPANIES_AUTHENTICATION_CODE_REMOVE_URL,
+            matomoButtonClick: constants.MATOMO_BUTTON_CLICK,
+            matomoLinkClick: constants.MATOMO_LINK_CLICK,
+            matomoAddNewAuthorisedPersonButton: constants.MATOMO_ADD_NEW_AUTHORISED_PERSON_BUTTON,
+            removeAuthorisedUser: constants.REMOVE_AUTHORISED_USER,
```

Comment:
The same as the above comment and also it would be good if you added the word link at the end of the variable name, so it would be `matomoRemoveAuthorisedUserLink`. Similar for the following.